contents/blog/using-vc-carefully.md

```diff
@@ -0,0 +1,117 @@
+---
+date: 2023-05-16
+title: VC is a tool, not a religion
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+category: CEO diaries
+tags:
+  - Marketing
+---
+
+We raised over $27,000,000 then realized we should be growing out of our own revenue instead.
+
+## Day 1 - we started bootstrapping, then changed our minds
+
+Fresh-faced, but not carefree - in the early days, when it was just my cofounder, Tim, and me, we used to work out of cafés and we were bootstrapped.
+
+A few months in, we had a discussion on what we wanted out of life. Pro tip - do this _before_ you start working with someone else. Luckily we were on the same page - we decided it was _more_ stressful to bootstrap than to raise VC.
+
+Why?
+
+* No money coming in meant we were having to be exceptionally careful about spending anything.
+* We'd spend years rebuilding to the same salaries that we'd just left behind, but with added downside risk of not finding product market fit.
+* A slower pace felt like it'd be boring. We'd find it more engaging to own a smaller slice of a bigger pie. We wanted to work on something we'd enjoy.
```

Comment:
```suggestion
* No money coming in meant we had to be exceptionally careful about spending anything.

* We would spend years rebuilding to the same salaries we'd just left behind, but with added downside risk of not finding product-market fit.

* A slower pace felt boring – it would be more engaging to own a smaller slice of a bigger pie. We wanted to work on something we'd enjoy.
```